spec/features/user_views_index_page_spec.rb

```diff
@@ -0,0 +1,34 @@
+# require "spec_helper"
+#
```

Comment:
This code is all commented out and should not be in the pull request. You don't want any code you're not actually using in your PRs.